app/javascript/react/components/PizzaShowContainer.js

```diff
@@ -0,0 +1,53 @@
+import React, { useState, useEffect } from 'react'
+
+const PizzaShowContainer = (props) => {
+  const [pizza, setPizza] = useState({ 
+    product_name: "", 
+    cost: null
+  })
+
+  const [brand, setBrand] = useState({})
+     
+  let pizzaId = props.match.params.id
+
+  const fetchPizza = async () => {
+    try {
+      const response = await fetch(`/api/v1/pizzas/${pizzaId}`)
+      if (!response){
+        const errorMessage = `${response.status} (${response.statusTest})`
+        const error = new Error(errorMessage)
+        throw(error)
+      }
+      const parsedPizza= await response.json()
+      setPizza(parsedPizza['pizza'])
+      setBrand(parsedPizza['brand'])
```

Comment:
Should be able to sue dot notation here instead of bracket, which would be consistent with your other React